File: src/Modules/Gambling/75+.cs
using Discord.Commands;
using System.Threading.Tasks;
using DEA.Common;

namespace DEA.Modules.Gambling
{
    public partial class Gambling : Module
    {
        [Command("75+")]
        [Remarks("50")]
        [Summary("Roll 75.01 or higher on a 100.00 sided die, win 2.8X your bet.")]
        public Task X3dot8(decimal bet)
        {
            return _gameService.GambleAsync(Context, bet, 75.01m, 2.8m);
        }
    }
}

File: src/commands/gambling/75+.js
const patron = require('patron.js');
const gambling = require('../../services/GamblingService.js');

class SeventyFivePlus extends patron.Command {
  constructor() {
    super({
      name: '75+',
      group: 'gambling',
      description: 'Roll 75.00 or higher on a 100.00 sided die to win 2.6X your bet.',
      args: [
        new patron.Argument({
          name: 'bet',
          key: 'bet',
          type: 'float',
          example: '500'
        })
      ]
    });
  }

  async run(context, args) {
    return gambling.gamble(context, args.bet, 75, 2.6);
  }
}

module.exports = new SeventyFivePlus();
